root/etc/s6-overlay/s6-rc.d/init-unifi-network-application-config/run

```diff
@@ -29,6 +29,16 @@ if [[ ! -e /config/data/system.properties ]]; then
         sed -i "s/~MONGO_PORT~/${MONGO_PORT}/" /defaults/system.properties
         sed -i "s/~MONGO_DBNAME~/${MONGO_DBNAME}/" /defaults/system.properties
         sed -i "s/~MONGO_PASS~/${MONGO_PASS}/" /defaults/system.properties
+        if [[ "${MONGO_TLS}" = "true" ]]; then
```

Comment:
Use `${MONGO_TLS,,}` so you don't run into case-sensitivity issues.

Reply:
Done.